src/screens/Instructors/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import {filter, reject} from 'lodash'
+import {instructorPulseTitleText, unpublishedTitleText, publishedTitleText} from 'utils/text'
+import Split from 'components/Split'
+import sortBy from 'sort-by'
+
+import InstructorsList from './components/InstructorsList'
+
+export default ({instructors}) => (
+  <Split
+    title={instructorPulseTitleText}
+    main={
+      <InstructorsList
+        title={unpublishedTitleText}
+        instructors={filter(instructors, ['published_lessons', 0])
+          .sort(sortBy('-submitted_lessons', '-claimed_lessons', '-approved_lessons', '-id'))}
+      />
+    }
+    aside={
+      <InstructorsList
+        title={publishedTitleText}
+        instructors={reject(instructors, ['published_lessons', 0])
+          .sort(sortBy('-submitted_lessons', '-claimed_lessons', '-approved_lessons', 'published_lessons', '-id'))}
```

Comment:
It is really amazing what sensible sorting can do for a dataset. This opened it up and made it super obvious who we needed to be helping. This list is actually my favorite part of this release <3 (aside from the foundation for so much future awesome)